projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -0,0 +1,113 @@
+import { Gridicon } from '@automattic/jetpack-components';
+import { Popover } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from 'react';
+import { PRODUCT_STATUSES } from '../../../constants';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import { timeSince } from '../../../utils/time-since';
+
+import './style.scss';
+
+const ProtectValueSection: React.FunctionComponent = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { status } = detail;
+	const isProtectActive =
+		status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
+
+	return isProtectActive ? <WithProtectValueSection /> : <NoProtectValueSection />;
+};
+
+export default ProtectValueSection;
+
+const WithProtectValueSection: React.FunctionComponent = () => {
+	const { protectStatus } = getMyJetpackWindowInitialState();
+	const lastScanTime = protectStatus?.last_checked;
+	const timeSinceLastScan = lastScanTime ? timeSince( Date.parse( lastScanTime ) ) : '...';
+
+	const lastScanText = sprintf(
+		/* translators: %s is how long ago since the last scan took place, i.e.- "17 hours ago" */
+		__( 'Last scan: %s', 'jetpack-my-jetpack' ),
+		timeSinceLastScan
+	);
+	return <ValueSection isProtectActive={ true } lastScanText={ lastScanText } />;
+};
+
+const NoProtectValueSection: React.FunctionComponent = () => {
+	const { plugins, themes } = getMyJetpackWindowInitialState();
+	const pluginsCount = Object.keys( plugins ).length;
+	const themesCount = Object.keys( themes ).length;
+
+	const pluginsThemesText = sprintf(
+		/* translators: %1$d is the number (integer) of plugins and %2$d is the number (integer) of themes the site has. */
+		__( '%1$s plugins & %2$s themes', 'jetpack-my-jetpack' ),
+		pluginsCount,
+		themesCount
+	);
+
+	return <ValueSection isProtectActive={ false } lastScanText={ pluginsThemesText } />;
+};
+
+const ValueSection: React.FunctionComponent< {
+	isProtectActive: boolean;
+	lastScanText: string;
+} > = ( { isProtectActive, lastScanText } ) => {
+	const isMobileViewport: boolean = useViewportMatch( 'medium', '<' );
+	const [ isVisiblePopover_PluginsThemes, setIsVisiblePopover_PluginsThemes ] = useState( false );
+
+	const togglePopover_PluginsThemes = function () {
+		setIsVisiblePopover_PluginsThemes( state => ! state );
```

Comment:
Nitpick, I like naming the state something more descriptive. I know the function makes it obvious what its doing but I think `previouslyVisible => ! previouslyVisible` or even just `prevState => ! prevState` would be a bit more accurate

Reply:
We'll also want to add a tracks event here to track the popover opens and closes

Reply:
Yes fair enough, sounds good! 🙂

> We'll also want to add a tracks event here to track the popover opens and closes

Yeah, my thought was to wait until I add the other popovers, to add the tracks events.  There may end up being a way to combine the tracks events for all 3 popovers into one function or something. When all 3 popovers are in place, that will be easier to determine, as well as maybe easier to determine the tracks event names and so forth.